uiHeader.R

```diff
@@ -0,0 +1,32 @@
+library(shiny)
+
+appTitle_ <- "MetaXcan Association results (257 phenotypes)"
```

Comment:
Although this syntax looks pleasant to the eye, I refrain from global variables whenever possible.
This app uses global variable only when itś the lesser evil, and I am not sure this is so in this case.